### PR TITLE
chore(tests): fix full_node_test

### PR DESCRIPTION
**Pull request by kstdl**
## Purpose

Fix some of fails in a full_node_test including accidentally changed `save_network_to_file test`

### CODE REVIEW

File: tests/full_node_test.cpp [hunk 1]
```diff
@@ -1176,7 +1176,7 @@ TEST_F(FullNodeTest, two_nodes_run_two_transactions) {
 
 TEST_F(FullNodeTest, save_network_to_file) {
   auto node_cfgs = make_node_cfgs(3);
-  auto nodes = launch_nodes(node_cfgs);
+  { auto nodes = launch_nodes(node_cfgs); }
```
Comment (JakubFornadel) [inline tests/full_node_test.cpp:1179]: what is this nodes even good for ? Seems it is not used in whole test ?

Comment (MatusKysel) [reply tests/full_node_test.cpp:1179]: here you create network and destroy it so next time will be loaded from file

Comment (MatusKysel) [reply tests/full_node_test.cpp:1179]: I do not know when the scope was removed as on develop it still there https://github.com/Taraxa-project/taraxa-node/blob/develop/tests/full_node_test.cpp#L1179

Comment (kstdl) [reply tests/full_node_test.cpp:1179]: It was accidentally removed here https://github.com/Taraxa-project/taraxa-node/pull/1044/files#diff-cb712a3395b4b23506c8277c7056adf0f3cf679e3e7f04bb3de00f3fb1a68dd4L1179. 
So just I returned it. You can also read it in the PR description :)

Comment (JakubFornadel) [reply tests/full_node_test.cpp:1179]: OK I get it now, although without your comments I would be totally lost on what is actually going on on this line - I dont know maybe a comment would be nice but that might be just me

**Review by MatusKysel — APPROVED** [review]

**Pull request merged**